.github/workflows/pull-request.yml

```diff
@@ -32,7 +41,7 @@ jobs:
           cache: 'npm'
 
       - name: Install NPM packages
-        run: npm ci
+        run: npm ci --no-audit --no-fund
```

Comment:
If we want to streamline these flags, I *think* the GH Actions will respect entries in a local `.npmrc` file

```
audit=false
fund=false
```

Reply:
I would rather keep those `ci` only. I do think that for every environment besides CI fund flags and audit reports are important.